src/screens/OrganizationPeople/AddMember.tsx

```
@@ -77,6 +74,28 @@
  *
  * For more details on the reusable classes, refer to the global CSS file.
  */
+
+interface Edge {
+  cursor: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)